### DISCUSSION TITLE
How to order pages in a collection in reverse chronological order

### DISCUSSION THREAD

**Discussion by ktatar** [alshedivat/al-folio#2720] [Q&A] [answered]
Hello, 

Is there a way to list the page cards under the projects collection in reverse chronological order? I know that I could use the importance: X option, but it is a bit tedious. When I add a new page, then I need to increase the number of importance parameter of all other pages to add a page to importance: 1. 

I also don't want to use the blog collection, since I would like to keep the categories in projects tab. 

So, is there any way to use the date of each project page to list them in reverse chronological order? 
Here is the page: https://aicomparts.com/projects/

Thanks in advance,

**Comment by george-gca** [comment, marked as answer]
By default, two documents in a collection are sorted by their `date` attribute when both of them have the `date` key in their front matter, as written in the [docs](https://jekyllrb.com/docs/collections/#custom-sorting-of-documents). Also, we already do sort a collection in an inverse order when generating the news part in the about page, so you just need to do the same for projects:

https://github.com/alshedivat/al-folio/blob/bdf4ce32e53bc9b4be1d1e10b796bd179cd87474/_includes/news.liquid#L11
